### PR TITLE
[cf.js] docs fixes around ID

### PR DESCRIPTION
**Pull request by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll**
various fixes to the docs related to `AppInstanceInfo.id` and the `appInstanceId` parameters

### CODE REVIEW

**Comment by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll** [comment]
hmm I actually wanted @mvanderh and @cf19drofxots to review for factual accuracy. Am about 90% sure the changes are correct

File: packages/cf.js/API_REFERENCE.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Params:
 
 Result:
 - `appInstanceId: string`
-    - Unique ID for this app instance
+    - ID of an app instance whose proposed installation was accepted
```
Comment (mvanderh) [inline packages/cf.js/API_REFERENCE.md]: @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll This isn't true. The installation isn't yet accepted when this ID is returned

**Comment by mvanderh** [comment]
Thanks @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll . I just saw one potential inaccuracy

File: packages/cf.js/API_REFERENCE.md (hunk 1 above)
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [reply packages/cf.js/API_REFERENCE.md]: Ah yeap, this should be changed

**Pull request merged**